### PR TITLE
Changing random_intergers() to randint()

### PR DESCRIPTION
**Pull request by mhnazeri**
np.random.random_integers() is depricated in numpy 1.11, we better use np.random.randint() instead.

### CODE REVIEW

File: intro/numpy/operations.rst [hunk 1]
```diff
@@ -404,7 +404,7 @@ Other reductions
   .. sourcecode:: pycon
 
    >>> t = np.arange(t_max)
-   >>> steps = 2 * np.random.random_integers(0, 1, (n_stories, t_max)) - 1
+   >>> steps = 2 * np.random.randint(0, 1 + 1, (n_stories, t_max)) - 1 # +1 because the high value is exclusive
```
Comment (gertingold) [inline intro/numpy/operations.rst]: +1, but wouldn't it be better to set the second argument directly to 2?

Comment (mhnazeri) [inline intro/numpy/operations.rst]: I think for clarification it is better to show it by +1 in the first example.

**Comment by mhnazeri** [comment]
I'm going through the tutorial from the beginning and running examples, I will amend this PR as soon as I encounter another randint()

**Comment by gertingold** [comment]
As far as I can see, `random_integers` is present in the following files

- advanced/image_processing/index.rst
- intro/numpy/operations.rst (you have taken care of this one already)
- intro/numpy/array_object.rst
- pyplots/numpy_intro_5.py

**Comment by mhnazeri** [comment]
_Thanks_. I've changed all those files.

**Comment by GaelVaroquaux** [comment]
Thanks for the pull request and the review. Merging

**Comment by gertingold** [comment]
Thanks for your contribution and sorry for looking into this so late. I am still not completely convinced by the explicit +1 in the random walk example because `randint` works like the familiar `range`, so the reader not knowing `random_integers` should not be too surprised. In addition, the correctness of the generated numbers is checked in the next code line. However, in view of #276 it seems appropriate to merge now and possibly amend the change later. 

@GaelVaroquaux: what do you think?

**Comment by GaelVaroquaux** [comment]
@gertingold : I agree with you and just merged.

**Comment by gertingold** [comment]
But what about the +1? Should we leave it as it is or assume that the reader will figure it out if `high` is set to 2?

**Comment by GaelVaroquaux** [comment]
I thought that we could leave it like this.

**Comment by gertingold** [comment]
Ok, fine.

**Pull request merged**